ui/src/views/infra/network/IpRangesTabGuest.vue

```diff
@@ -137,7 +137,7 @@ export default {
     this.fetchData()
   },
   watch: {
-    network (newItem, oldItem) {
+    resource (newItem, oldItem) {
```

Comment:
@davidjumani With Vue3, to watch an Object, you should use deep. Refer: https://vuejs.org/guide/essentials/watchers.html#deep-watchers

Reply:
@utchoang Doc says `deep` is needed only in cases where trying to run callback on all nested mutations. Even there example runs fine without it and I don't think we want to run `fetchData` here on any property change of `resource`